ecosystem/sep-0010.md

```diff
@@ -121,6 +126,12 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+### Memos
+
+Stellar transactions may be specified with a memo and services that share a single account for many users typically use the memo as a method to distinguishing one user from another, or one internal off-network account from another.
+
+The `memo_type` and `memo` parameters supported in `GET <WEB_AUTH_ENDPOINT>` API calls can be used to communicate to the server that the client intends to authenticate a Stellar account with an additional claim that the authenticating user has control of or is possession of the provided `memo`.
+
```

Comment:
Can we move this to the bottom of the document after the `JWT Expiration` section?